src/api/v1/server.go

```diff
@@ -19,12 +20,17 @@ func API(api *echo.Group) {
 	api.GET("/minecraft/user/info", getUserInfo, middleware.CacheUntilPurge())
 	api.GET("/minecraft/user/:role/list", getRoleMembers, middleware.CacheUntilPurge())
 	api.GET("/dbtest", dbTest, middleware.NoCache())
-	api.Match([]string{http.MethodGet, http.MethodPost}, "/login/password", jwt.PasswordLoginHandler, middleware.NoCache())
-	api.Match([]string{http.MethodGet, http.MethodPost}, "/login/minecraft", jwt.MinecraftLoginHandler, middleware.NoCache())
-	api.Match([]string{http.MethodGet, http.MethodPost}, "/login/discord", jwt.DiscordLoginHandler, middleware.NoCache())
+	api.GET("/user/me", getUser, middleware.NoCache(), middleware.RequireAuth(), middleware.Limit(10*time.Microsecond, 5))
+	api.PATCH("/user/me", patchUser, middleware.NoCache(), middleware.RequireAuth(), middleware.Limit(50*time.Millisecond, 5))
+	api.PUT("/password/me", putPassword, middleware.NoCache(), middleware.RequireAuth(), middleware.Limit(50*time.Millisecond, 5))
+	api.PUT("/password/:token", putPassword, middleware.NoCache(), middleware.Limit(50*time.Millisecond, 5))
+	api.Match([]string{http.MethodGet, http.MethodPost}, "/password/reset", resetPassword, middleware.NoCache(), middleware.Limit(100*time.Millisecond, 5))
+	api.Match([]string{http.MethodGet, http.MethodPost}, "/login/password", jwt.PasswordLoginHandler, middleware.NoCache(), middleware.Limit(10*time.Millisecond, 5))
```

Comment:
you're actually out of your mind LMAO

fuck password limits, we allowin ONE HUNDRED ATTEMPTS PER SECOND PER IP

Reply:
What's a sensible limit? Maybe this is where the burst number comes into play; you can make an attempt every 10ms but you can only do that 5 times in a row without a cool down? IDK

Reply:
these should be up in the seconds imo

Reply:
what does "in a row" mean? i just need to alternate with another request to get around it....?

Reply:
I've reverted the ratelimit stuff for now, we can look at that after merging.